src/google/protobuf/metadata.h

```diff
@@ -59,8 +59,7 @@ namespace internal {
 class LIBPROTOBUF_EXPORT InternalMetadataWithArena {
```

Comment:
I don't think so, because then the class wouldn't be exported from the DLL, which means other code trying to link against the protobuf DLL wouldn't be able to access this (and we can tell such code wants to access this from the error I describe above).

If we want consistency across the class we could de-inline the whole thing, if that won't cause a performance problem.

I also feel like I tried other solutions, e.g. __declspec(selectany), and they didn't work, but I can't quite remember.